docs/evaluate/development-features/schedules.mdx

```diff
@@ -0,0 +1,39 @@
+---
+id: schedules
+title: Schedules - Temporal Development Feature
+description: Discover how to automate and control your Workflow executions with Temporal's scheduling capabilities. Learn the benefits of scheduling Workflows and explore best practices to ensure timely and efficient execution of your business processes.
```

Comment:
I vote to lose the first sentence because we aren't controlling the executions themselves. I think just the second sentence is enough! :)